### PR TITLE
Added energy efficiency img README.md

### PR DESCRIPTION
**Pull request by Aliki92**
##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ It scales nicely from just a single server to thousands of servers, even in comp
   Netdata is a modular platform that can be extended in all possible ways and it also integrates nicely with other monitoring solutions.
 
 ---
-![Energy-efficiency 1 (1)](https://github.com/netdata/netdata/assets/139226121/cc44aa1e-3e96-42ec-87d0-ee10d711ce74)
+![Energy-efficiency-img](https://github.com/netdata/netdata/assets/139226121/cc44aa1e-3e96-42ec-87d0-ee10d711ce74)
```
Comment (ilyam8) [inline README.md:66]: Can this by a hyperlink to the research?

**Review by Ancairon — COMMENTED** [review]
@Aliki92 this is not working with incognito because the images are stored in the website which is a private repo, we need to store them somewhere in this one

File: README.md [hunk 2]
```diff
@@ -63,6 +63,14 @@ It scales nicely from just a single server to thousands of servers, even in comp
   Netdata is a modular platform that can be extended in all possible ways and it also integrates nicely with other monitoring solutions.
 
 ---
+<p align="center">
+<a href="https://www.ivanomalavolta.com/files/papers/ICSOC_2023.pdf#gh-light-mode-only">
+  <img src="https://www.netdata.cloud/img/readme-images/Energy-efficiency.jpg" alt="Energy Efficiency" width="850"/>
```
Comment (tkatsoulas) [inline README.md:68]:
```suggestion
  <img src="https://www.netdata.cloud/img/readme-images/Energy-efficiency%20-%20light.jpg" alt="Energy Efficiency" width="850"/>
```

File: README.md [hunk 3]
```diff
@@ -63,6 +63,14 @@ It scales nicely from just a single server to thousands of servers, even in comp
...
+</a>
+<a href="https://www.ivanomalavolta.com/files/papers/ICSOC_2023.pdf#gh-dark-mode-only">
+  <img src="www.netdata.cloud/img/readme-images/Energy-efficiency-light.jpg" alt="Energy efficiency" width="850"/>
```
Comment (ilyam8) [inline README.md:71]:
```suggestion
  <img src="https://www.netdata.cloud/img/readme-images/Energy-efficiency-light.jpg" alt="Energy efficiency" width="850"/>
```

File: README.md [hunk 4]
```diff
@@ -63,6 +63,14 @@ It scales nicely from just a single server to thousands of servers, even in comp
   Netdata is a modular platform that can be extended in all possible ways and it also integrates nicely with other monitoring solutions.
 
 ---
+<p align="center">
+<a href="https://www.ivanomalavolta.com/files/papers/ICSOC_2023.pdf#gh-light-mode-only">
+  <img src="https://www.netdata.cloud/img/readme-images/Energy-efficiency.jpg" alt="Energy Efficiency" width="850"/>
+</a>
+<a href="https://www.ivanomalavolta.com/files/papers/ICSOC_2023.pdf#gh-dark-mode-only">
```
Comment (ilyam8) [inline README.md:70]:
```suggestion
<a href="https://www.ivanomalavolta.com/files/papers/ICSOC_2023.pdf#gh-dark-mode-only">
  <img src="https://www.netdata.cloud/img/readme-images/Energy-efficiency.jpg" alt="Energy Efficiency" width="850"/>
</a>
<a href="https://www.ivanomalavolta.com/files/papers/ICSOC_2023.pdf#gh-light-mode-only">
```

**Pull request merged**